### ISSUE TITLE
[Visual Requirements – Make Code - Review, Commit and push local changes to GitHub] Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3766]
User Experience:
Visual impaired user would find difficulty to see the control or element if that controls have insufficient color contrast as they won't be able to take on that controls and he/she would miss the functionality of such control or element.

Note: User credentials should NOT be included in the bug.

Repro Steps:

1. Launch application URL
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Review, Commit and push local changes to GitHub" button and select it.
6. Navigate through all the controls appearing.
7. Verify the issue

Actual Result:
Luminosity ratio is less than 4.5:1 for text control

Expected Result:
Luminosity ratio should be more than and equal to 4.5:1 for text control 

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://dequeuniversity.com/rules/axe/4.0/color-contrast?application=msftAI)

Target application:  Microsoft MakeCode for micro:bit - https://makecode.microbit.org/#editor

Element path: .repo-branch

Snippet:

<span role="button" class="repo-branch">#master<i class="dropdown icon"></i></span>

How to fix:
Fix any of the following:
Element has insufficient color contrast of 2.36 (foreground color: #a8a8a8, background color: #ffffff, font size: 15.4pt (20.5714px), font weight: bold). Expected contrast ratio of 3:1

Environment: Microsoft Edge version 88.0.692.0

Note:
Same issue exists on Firefox & Chrome.

WCAG Reference:
[WCAG 1.4.3 - Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: Accessibility insights for web
Screen Reader: Narrator, NVDA, JAWS

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #HCL-MakeCode-New; #WCAG1.4.3; #Win10-Edge(Chromium);

![WCAG1 4 3_Master](https://user-images.githubusercontent.com/72871089/100226390-b02e3800-2f45-11eb-9333-fdf77c05b659.png)
